packages/commonwealth/server/controllers/server_comments_methods/delete_comment.ts

```diff
@@ -16,15 +16,31 @@ export type DeleteCommentOptions = {
   user: UserInstance;
   address: AddressInstance;
   chain: ChainInstance;
-  commentId: number;
+  commentId?: number;
+  message_id?: string;
 };
 
 export type DeleteCommentResult = void;
 
 export async function __deleteComment(
   this: ServerCommentsController,
-  { user, address, chain, commentId }: DeleteCommentOptions
+  { user, address, chain, commentId, message_id }: DeleteCommentOptions
 ): Promise<DeleteCommentResult> {
```

Comment:
While the query params use `snake_case`, the properties passed to the method should be `camelCase`